src/View/ViewVarsTrait.php

```diff
@@ -157,6 +157,7 @@ public function set($name, $value = null)
      * @param bool $merge Whether to merge with or override existing valid View options.
      *   Defaults to `true`.
      * @return array The updated view options as an array.
+     * @deprecated 3.7.0 Use ViewBuilder::setOptions() or any one of it's setter methods instead.
      */
     public function viewOptions($options = null, $merge = true)
```

Comment:
Should this method emit a deprecation warning?

Reply:
Yup, will do

Reply:
Done